src/SubNav.js

```diff
@@ -25,13 +25,28 @@ const SubNav = styled(SubNavBase)`
   .SubNav-body {
     display: flex;
     margin-bottom: -1px;
+
+    > * {
+      margin-left: ${get('space.2')};
+    }
+
+    > *:first-child {
+      margin-left: 0;
+    }
   }
 
   .SubNav-actions {
     align-self: center;
   }
 
   ${COMMON};
+
+  ${FLEX};
+`
+
+SubNav.Links = styled.div`
```

Comment:
Sorry one more thing! I think this could just be:
```suggestion
SubNav.Links = Flex`
```

Reply:
Thanks! This is exactly the feedback I need 🙇